int8 port
int8 value
